sphinx/domains/python.py

```diff
@@ -1016,6 +1056,23 @@ def process_link(self, env: BuildEnvironment, refnode: Element,
         return title, target
 
 
+class PyPropXRefRole(PyXRefRole):
+    """A special xref role class for :py:prop: role.
+
+    It adds a "py-attr" style to the role for backward compatibility.
+
+    Note: RemovedInSphinx50Warning
```

Comment:
@flying-sheep: This isn't about backwards-compatibility of _sphinx_, right? It's about backwards compatibility of libraries changing between `attr` and `prop` - so I don't think it makes sense to ever deprecate it.

Reply:
Yes, it’s about backwards compat of themes and custom css, and it doesn’t make sense to remove it. Most themes will probably forever style both kinds of links the same way.

In fact, I can’t think of a reason not to style the links the same way.